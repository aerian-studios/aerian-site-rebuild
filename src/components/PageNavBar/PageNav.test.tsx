/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

// import { PageNavBar } from "./index";

describe("PageNavBar", () =>
    xit("renders correctly", () => {
        const tree = renderer
            .create(<PageNavBar className="myClass" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    }));
