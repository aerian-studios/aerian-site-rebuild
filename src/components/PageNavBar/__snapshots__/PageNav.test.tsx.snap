// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNavBar renders correctly 1`] = `
<header
  className="component myClass"
>
  <nav
    className="component "
    id="menu-main"
  />
</header>
`;
